thirdparty/grpc/fix-constants-on-windows.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/third_party/upb/upb/message/internal/message.c b/third_party/upb/upb/message/internal/message.c
+index 57ab2d7371..5f321c7bb2 100644
+--- a/third_party/upb/upb/message/internal/message.c
++++ b/third_party/upb/upb/message/internal/message.c
```

Comment:
```
D:\a\nifi-minifi-cpp\nifi-minifi-cpp\build\_deps\grpc-src\third_party\protobuf\upb\message\internal\message.c(22,14): error C2099: initializer is not a constant
const double kUpb_NaN = NAN;
```
The path seems different in the CI failure that happened after this patch was added.